README.md

```diff
@@ -17,80 +17,19 @@
 
 <div>&nbsp;</div>
 
-## Installation
-**Roact-Rodux expects to be located inside the same object as [Roact](https://github.com/Roblox/Roact) and [Rodux](https://github.com/Roblox/Rodux). They should be installed into the same place!**
+Roact-Rodux is an ergonomic binding [Roact](https://github.com/Roblox/roact), a view library, and [Rodux](https://github.com/Roblox/rodux), a state management library.
```

Comment:
Are you using ergonomic as a noun here?  Kind of unusual language, imo.  Did you mean "ergonomic binding between"? Or maybe just the exact sentence you use in the docs index.md